iseus/styles.css

```diff
@@ -180,9 +180,9 @@ section h2{
 }
 .sec2{
```

Comment:
Try to use one style for similar sections. Or you can simply declare another style that includes the common style. Please note that a tag (e.g. div) can use multiple classes. (e.g. classes="news blur-image round-border sm-shadow"). As you can see in the example, I can use the news class for all my news sections and change the unique one with giving it extra classes line round-border or etc.